backend/src/common/pipes/validation.pipe.ts

```diff
@@ -0,0 +1,25 @@
+import { HttpException } from '@nestjs/core';
+import { PipeTransform, Pipe, ArgumentMetadata, HttpStatus } from '@nestjs/common';
+import { validate } from 'class-validator';
+import { plainToClass } from 'class-transformer';
+
+@Pipe()
+export class ValidationPipe implements PipeTransform<any> {
```

Comment:
`ValidationPipe` too - should we leave it as an example or switch to the built-in one?